libs/cardano-ledger-pretty/cardano-ledger-pretty.cabal

```
@@ -39,7 +39,7 @@ library
         cardano-ledger-alonzo >=1.2,
         cardano-ledger-babbage >=1.1,
         cardano-ledger-byron,
-        cardano-ledger-conway ^>=1.7,
+        cardano-ledger-conway >=1.7,
```

That bound is invalid. You made a breaking change that `cardano-ledger-pretty` now depends on.
```suggestion
        cardano-ledger-conway ^>=1.8,
```

---

Oh yes! What I did is wrong :+1: Thanks!